docs/SOP.md

```diff
@@ -39,3 +40,48 @@ In general, the metadata record of an ontology in the OBO Foundry metadata regis
 7. If at least 3 months have passed after the initial request for feedback, and the above conditions are met, any OBO Foundry operations committee member MAY merge the pull request at their own discretion.
 
 For a discussion on this SOP, see [here](https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1848).
+
+<a name="OPS_CHAIR"></a>
+
+## Chairing an OBO Operations Committee meeting call
+
+### Before the call (a day or two in advance):
+1. Prepare a stub in the OBO Operations Committee (OFOC) [rolling agenda](https://docs.google.com/document/d/1aka4i6R89i04IYPS7CyzItQPOyb3IgtW4m75G475qcc/edit) with the following <b>Repeating Agenda Items</b>:
+
+     1. Check for [pending members for obo-discuss](https://groups.google.com/g/obo-discuss/pending-members) 
+     2. Get volunteers to sign up to lead upcoming meetings (if needed)
+     3. Review [new ontology requests](https://github.com/OBOFoundry/OBOFoundry.github.io/labels/new%20ontology)
+     4. Report from Editorial Working Group (EWG) (Darren)
+     5. Report from Technical Working Group (TWG) (Nico)
+     6. Review additional [open issues](https://github.com/OBOFoundry/OBOFoundry.github.io/labels/attn%3A%20OFOC%20call) and [pull requests](https://github.com/OBOFoundry/OBOFoundry.github.io/pulls?q=is%3Apr+is%3Aopen+label%3A%22attn%3A+OFOC+call%22) that are labeled “attn: OFOC call”
+
+2. Check the issues labeled [“attn:OFOC call”](https://github.com/OBOFoundry/OBOFoundry.github.io/labels/attn%3A%20OFOC%20call) found at the OBO foundry github repository. Pick one or two open issues you deem important and put them towards the end of the stub agenda, after the Working Group reports and before the review of additional open issues.
```

Comment:
Quotes look like they are wrongly formatted. Should be `"`, not `“`.

Reply:
Happens in a few other places

Reply:
We edited the original after copying to a Google doc to facilitate group editing. These are a holdover of that process. Will correct them.